README.md

```diff
@@ -18,7 +18,8 @@ Instructions are provided for deployment through GitHub Codespaces, VS Code Dev
 
 #### Architecture diagram
 
-![Architecture diagram showing that user input used in conjunction with user identity to call app code running in Azure Container apps that processes the user input and generates a response to the user. The app code leverages Azure AI projects, Azure AI model inference, prompty, and Azure AI Search.](docs/architecture.png)
+![Architecture diagram showing that user input is provided to the Azure Container App, which contains the app code. With user identity and resource access through managed identity, the input is used to form a response. The input and the Azure monitor are able to use the Azure resources deployed in the solution: Application Insights, Azure AI Project, Azure AI Services, Azure AI Hub, Storage account, container apps environment, Container Registry, Key Vault, and Log Analytics Workspace.](docs/architecture.png)
```

Comment:
"container apps environment" -> Azure Container App

To follow the style as with other resources listed